### PR TITLE
Initial commit for Serratus lovelywater

### PR DESCRIPTION
**Pull request by chueatwork**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/serratus-lovelywater.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Name: Serratus Coronavirus Discovery - Versioned Data Release
```
Comment (ababaian) [inline datasets/serratus-lovelywater.yaml:1]: Option for more general use sub-title, "Serratus:  Ultra-deep Search for Novel Viruses -  Versioned Data Release"

File: datasets/serratus-lovelywater.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Description: "Serratus is a collaborative open science project for ultra-rapid discovery of known and unknown coronaviruses in response to the COVID-19 pandemic through re-analysis of publicly available genomic data. Our resulting vertebrate viral alignment data is explorable via the [Serratus Explorer](serratus.io/explorer) and directly accessible on Amazon S3."
+Documentation: https://github.com/ababaian/serratus/wiki/Access-Data-Release
+Contact: https://github.com/ababaian/serratus/issues
+ManagedBy: Serratus
+UpdateFrequency: Occasionally
```
Comment (ababaian) [inline datasets/serratus-lovelywater.yaml:6]: "Quarterly" is the goal

**Review by ababaian — APPROVED** [review]

File: datasets/serratus-lovelywater.yaml (hunk 2 above)
Comment (chueatwork) [reply datasets/serratus-lovelywater.yaml:6]: Got it, thanks!

File: datasets/serratus-lovelywater.yaml (hunk 1 above)
Comment (chueatwork) [reply datasets/serratus-lovelywater.yaml:1]: I like this much better; changing now!

**Comment by ababaian** [comment]
Oooh sorry I should also mention, might be worth linking the Mammalian Deltavirus paper from our colleagues.

[ Bergner et al., 2021](https://www.pnas.org/content/118/3/e2019907118)

**Pull request merged**